saritasa_invocations/k8s.py

```diff
@@ -0,0 +1,222 @@
+    config = _config.Config.from_context(context)
+    env = env or config.default_k8s_env
+    environment = get_environment(context, env)
+    context.run(f"kubectl config use-context {environment.cluster}")
+    context.run(
+        f"kubectl config set-context "
+        f"--current --namespace={environment.namespace}",
+    )
+    printing.print_success(
+        f"`{environment.name}` environment has been set up successfully.",
+    )
+
+
+@invoke.task
+def login(context: invoke.Context) -> None:
+    """Login into k8s via teleport."""
+    printing.print_success("Login into kubernetes CI")
+    config = get_current_env_config_from_context(context)
+    context.run(
+        "tsh login"
+        f" --proxy={config.proxy}"
+        f" --auth={config.auth}"
+        f" --kube-cluster={config.proxy}",
+    )
+
+
+@invoke.task
+def logs(
+    context: invoke.Context,
+    component: str = "",
+) -> None:
+    """Get logs for k8s pod."""
+    config = get_current_env_config_from_context(context)
+    component = component or config.default_component
+    success(context, f"Getting logs from {component}")
+    context.run(
+        f"kubectl logs $({get_pod_cmd(context, component=component)})",
+    )
+
+
+@invoke.task
+def pods(context) -> None:
+    """Get pods from k8s."""
+    success(context, "Getting pods")
+    context.run("kubectl get pods")
+
+
+@invoke.task
+def execute(
+    context: invoke.Context,
+    entry: str = "",
+    component: str = "",
+    pty: bool | None = None,
+    hide: str | None = None,
+) -> invoke.Result | None:
+    """Execute command inside k8s pod."""
+    config = get_current_env_config_from_context(context)
+    component = component or config.default_component
+    entry = entry or config.default_entry
+    success(context, f"Entering into {component} with {entry}")
+    return context.run(
+        f"kubectl exec -ti $({get_pod_cmd(context, component)}) -- {entry}",
+        pty=pty,
+        hide=hide,
+    )
+
+
+@invoke.task
+def python_shell(
+    context: invoke.Context,
+    component: str = "",
+) -> None:
+    """Enter into python shell."""
+    config = get_current_env_config_from_context(context)
+    execute(context, component=component, entry=config.python_shell)
+
+
+@invoke.task
+def health_check(
+    context: invoke.Context,
+    component: str = "",
+) -> None:
+    """Check health of component."""
+    config = get_current_env_config_from_context(context)
+    execute(context, component=component, entry=config.health_check)
+
+
+def download_file_from_pod(
+    context: invoke.Context,
+    pod_namespace: str,
+    pod_command: str,
+    path_to_file_in_pod: str,
+    path_to_where_save_file: str,
+    retries: int = -1,
+) -> None:
+    """Download file from pod."""
+    context.run(
+        f"kubectl cp --namespace {pod_namespace} --retries={retries}"
+        f"$({pod_command}):{path_to_file_in_pod} "
+        f"{path_to_where_save_file}",
```

Comment:
Is it enough spaces?
```suggestion
        f" $({pod_command}):{path_to_file_in_pod}"
        f" {path_to_where_save_file}",
```